css/style.css

```diff
@@ -0,0 +1,24 @@
+#premium {
+  display: flex;
+  flex-direction: row;
+}
+#header {
+  position: absolute;
```

Comment:
you should change the header position to fixed, to avoid header problems advancing in the project.

Reply:
- [ ] Also, I recommend being more gentle in your reviews. Some developers feel stressed so a gentle review will help them feel better.